UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -1260,6 +1147,100 @@ protected virtual void LethalElectrocution(Electrocution electrocution, float sh
 		}
 
 		#endregion
+
+				/// <summary>
+		/// The list of the internal net ids of the body parts contained within this container
+		/// </summary>
+		[Tooltip("The internal net ids of the body parts contained within this")]
+		public List<uint> InternalNetIDs;
+
+
+		public RootBodyPartController rootBodyPartController;
+
+
+		/// <summary>
+		/// Sets up the sprite of a specified body part and adds its Net ID to InternalNetIDs
+		/// </summary>
+		/// <param name="implant">Body Part to display</param>
+		public void SetupSpritesNetId(BodyPart implant)
+		{
+			int i = 0;
+			bool isSurfaceSprite = implant.IsSurface;
+			var sprites = implant.GetBodyTypeSprites(playerSprites.ThisCharacter.BodyType);
+			foreach (var Sprite in sprites.Item2)
+			{
+				var newSprite = Spawn.ServerPrefab(implant.SpritePrefab.gameObject, Vector3.zero, playerSprites.BodySprites.transform).GameObject.GetComponent<BodyPartSprites>();
+				newSprite.transform.localPosition = Vector3.zero;
+				newSprite.name = implant.name;
+				playerSprites.Addedbodypart.Add(newSprite);
+
+
+				if (isSurfaceSprite)
+				{
+					playerSprites.SurfaceSprite.Add(newSprite);
+				}
+
+				implant.RelatedPresentSprites.Add(newSprite);
+
+				var newOrder = new SpriteOrder(sprites.Item1);
+				newOrder.Add(i);
+				newSprite.UpdateSpritesForImplant(implant, implant.ClothingHide, Sprite, newOrder);
+				InternalNetIDs.Add(newSprite.GetComponent<NetworkIdentity>().netId);
+
+				i++;
+				i++;
+				i++;
```

Comment:
```suggestion
				i += 3;
```
Increment thrice? If that's what you want, perhaps `i += 3` with a quick comment would look less like a mistake.